src/EnergyPlus/DXCoils.cc

```diff
@@ -131,9 +131,9 @@ using namespace ScheduleManager;
 // Functions
 
 void SimDXCoil(EnergyPlusData &state,
-               std::string_view CompName,     // name of the fan coil unit
-               int const CompOp,              // compressor operation; 1=on, 0=off
-               bool const FirstHVACIteration, // True when first HVAC iteration
+               std::string_view CompName,        // name of the fan coil unit
+               CompressorOperation const CompOp, // compressor operation; 1=on, 0=off
```

Comment:
I hate to be pedantic (actually, I love it) but I think `ComprOp` or `CompressOp` or `CompressorOp` is better than `CompOp`. We are already using `Comp` to mean component (see preceding parameter), we shouldn't also use it to mean compressor.